src/finfield.c

```diff
@@ -600,6 +600,8 @@ FF              FiniteField (
 
     /* enter the finite field in the tables                                */
 #ifdef HPCGAP
+    MakeBagReadOnly(bag1);
```

Comment:
bag1 is forgotten just a line or two later. It held temporary data used while constructing the SuccFF table and then discarded.

Reply:
I'll remove that line, and rename the local vars for clarity.